shared/css/professional_learning_marketing_banner.scss

```diff
@@ -1,250 +1,69 @@
-        margin-bottom: 8px;
-      }
-
-      .text {
-        color: black;
-        position: relative;
-        font-size: 14px;
-        width: calc(100% - 150px);
-        line-height: 1.3;
-      }
-    }
-
-    .right-button {
-      padding-top: 32px;
-      text-align: center;
-      position: absolute;
-      right: 64px;
-
-      button {
-        margin-top: 0px;
-        font-size: 16px;
-      }
-    }
-  }
-
-  .professional-learning-banner-tablet.homepage {
-
-    .text-wrapper {
-      right: 24px;
-
-      .header {
-        color: black;
-        padding-bottom: 5px;
-        font-size: 2.5vw;
-      }
-
-      .text {
-        color: black;
-        font-size: 2vw;
-      }
-
-      button {
-        margin-top: 10px;
-      }
-    }
+  @media screen and (max-width: $width-md) {
+    height: unset;
+    padding: 1.5rem;
+    text-align: center;
+    background-size: cover;
   }
 
-  .professional-learning-banner-tablet {
+  .wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+    flex-wrap: wrap;
+    max-width: 960px;
     margin: 0 auto;
-    position: relative;
-    width: 100%;
-
-    .banner {
-      text-align: center;
-      margin: 0 auto;
-      position: relative;
 
-      img {
-        width: 100%;
-      }
+    @media screen and (max-width: $width-lg) {
+      justify-content: end;
+      gap: 2rem;
     }
 
-    .text-wrapper {
-      right: 12px;
-      width: 52%;
-      float: right;
-
-      .header {
-        color: white;
-        padding: 15px 0px 5px 0px;
-        text-align: center;
-        top: 0px;
-        font-size: 16px;
-      }
-
-      .text {
-        color: white;
-        line-height: 1.2;
-        text-align: center;
-        font-size: 13px;
-      }
-
-      button {
-        font-size: 14px;
-        margin-bottom: 15px;
-      }
+    @media screen and (max-width: $width-md) {
+      flex-direction: column;
+      gap: 1rem;
     }
-  }
-
-  .professional-learning-banner-phone.homepage {
-    background-color: $gold_background_colour;
 
     .text-wrapper {
-      color: black;
+      width: 40%;
 
-      .header {
-        font-size: 18px;
+      h1, p {
+        text-shadow: 1px 2px 1px rgba(255, 223, 161, 0.95);
       }
 
-      .text {
-        font-size: 14px;
-      }
-    }
-  }
-
-  .professional-learning-banner-phone {
-    text-align: center;
-    position: relative;
-    width: 100%;
-    height: 180px;
-    background-color: $dark_blue_background_colour;
-
-    .text-wrapper {
-      color: white;
-      width: 100%;
-      padding-left: 24px;
-      padding-right: 24px;
-
-      .header {
-        padding: 15px 30px 5px 30px;
+      @media screen and (max-width: $width-lg) {
         text-align: center;
-        top: 0px;
-        font-size: 16px;
-      }
+        width: 60%;
 
-      .text {
-        text-align: center;
-        line-height: 1.3;
+        h1, p {
+          color: var(--neutral_white);
```

Comment:
Curious about this version of calling a color as opposed to the hash version above!

Reply:
This is one of the brand colors that has a `:root` variable ([ref](https://github.com/code-dot-org/code-dot-org/blob/613d710148803ab4f0149fd233005727d5c867ab/pegasus/sites.v3/code.org/styles_min/040-page.css#L10-L30)), but the other two at the top are specific only to this banner so I kept them as hex values.

Reply:
This makes sense. Thanks for explaining